salt/states/boto_rds.py

```diff
@@ -733,3 +733,77 @@ def parameter_present(name, db_parameter_group_family, description, parameters=N
         else:
             ret['comment'] = os.linesep.join([ret['comment'], 'Parameters {0} for group {1} are present.'.format(params, name)])
     return ret
+
+
+def db_cluster_present(name, engine, master_username, master_user_password, parameters=None,
```

Comment:
Since the norm for Salt states is to enforce a given configuration, whether when creating new resources or updating existing ones, I think it would be good to call out somewhere in this function that it is unable to UPDATE existing clusters -- if a change is required, the cluster would need to be destroyed and recreated.  This is in no way your fault and TBH, it will require porting this module entirely to boto3 to make such a thing possible, but I do think that documenting this atypical behaviour would serve the principal of least surprise in this case.